src/main/scala/org/constellation/consensus/Consensus.scala

```diff
@@ -58,6 +59,7 @@ class Consensus[F[_]: Concurrent](
         .map(_.map(_.transaction))
       messages <- Sync[F].delay(dao.threadSafeMessageMemPool.pull())
       notifications <- LiftIO[F].liftIO(dao.peerInfo.map(_.values.flatMap(_.notification).toSeq))
+      observations <- observationService.pullForConsensus(0)
```

Comment:
"0" can be the default if it's always zero or better a config variable with name

Reply:
It's `0` only for non-required things like observations (I want to pull something, without throwing an error if there are no objects). For transactions, it's configured in variable